### ISSUE TITLE
save template settings without changing colour - contrast colour is wrong

### ISSUE THREAD

**Issue by nadjak77** [joomla/backend-template#85]
### Steps to reproduce the issue

go to template settings and save without changing colours

### Expected result

atum-contrast-colour should be like turkis

### Actual result

the colour is a dark green without contrast to the dark blue

### System information (as much as possible)

I think, the color calculation is wrong.
Should be testet with new colorpicker if it is ready

**Comment by HRIT-Florian** [comment]
@nadjak77 I've tried it hard, but could NOT reproduce it! Tested with latest release branche.

bevore:
![bevore](https://user-images.githubusercontent.com/12803406/59566211-27775100-905d-11e9-8276-33277cd5aa3b.jpg)
after:

![after](https://user-images.githubusercontent.com/12803406/59566210-27775100-905d-11e9-9e36-0153df3b6b71.jpg)

The saved color params in the db looks also good.
`{"hue":"210","bg-light":"#e8e8e8","text-dark":"#495057","text-light":"#ffffff","link-color":"#4d78a3","special-color":"#001b4c","monochrome":"0","siteLogo":"","altSiteLogo":"","smallLogo":"","altSmallLogo":"","loginLogo":"","altLoginLogo":""}`

Maybe i misunderstood? Or where did you see the wrong color?
